testCases/test_atms.ipynb

```diff
@@ -21,7 +21,8 @@
     "\n",
     "<h3>Record of Revisions:</h3>\n",
     "    \n",
-    "Date: 2020-02-26 Author: P. Stegmann Description: Original code"
+    "Date: 2020-02-26 Author: P. Stegmann Description: Original code\\\n",
+    "&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2021-03-01 Author: B. Karpowicz Description: update API to use Salinity instead of S2m"
```

Comment:
Can this be removed?
```
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Reply:
What do you think of using a table?
[https://stackoverflow.com/questions/48655801/tables-in-markdown-in-jupyter](url)

Reply:
I like, it. I'll put one in.